_data/sources/meetup.js

```diff
@@ -1,17 +1,63 @@
-const fetch = require("node-fetch"),
+const {GraphQLClient, gql} = require("graphql-request"),
     group = require("./templates/group"),
     event = require("./templates/event");
 
 class meetup {
-    constructor() {
+    constructor(token = process.env.MEETUP_TOKEN) {
         this.groups = require("./groupIds/meetup").filter((value, index, self) => self.indexOf(value) === index);
-        this.apiEvents = (group) => 'https://api.meetup.com/' + group + '/events';
+        this.graphQLClient = new GraphQLClient('https://api.meetup.com/gql', {
+            headers: {
+                authorization: 'Bearer ' + token,
+                method: 'POST',
+                dataType: 'jsonp'
+            },
+        });
+        this.apiEvents = (group) => gql`
+            {
+                groupByUrlname(urlname: ${group}) {
+                    unifiedEvents {
+                        edges {
+                            node {
+                                title
+                                shortUrl
+                                venue
+                                description
+                                dateTime
+                                endTime
+                                going
+                                maxTickets
+                                fees
+                                price
+                                groupName
+                                groupLink
+                                source
+                                ad
+                            }
+                        }
+                    }
+                    name
+                    link 
+                }
+            }
+        `;
         this.apiPastEvents = (group) => 'https://api.meetup.com/' + group + '/events?desc=true&status=past';
-        this.apiGroup = (group) => 'https://api.meetup.com/' + group;
-        this.header = {
-            method: 'GET',
-            dataType: 'jsonp'
-        };
+        this.apiGroup = (group) => gql`
+            {
+                groupByUrlname(urlname: ${group}) {
+                    id
+                    name
+                    desc
+                    link
+                    links
```

Comment:
This property not only does not come from the API, but it is only properly populated during aggregation within the data gatherer script, this property is used to hold links within Objects such as the following: Meetup, Eventbrite, Facebook, Slack, Twitter, LinkedIn, etc.

Reply:
Uncertainly is expected, since I need to add proper documentation for this piece of functionality which was only added recently.

Reply:
This shall be documented in reference to #109.

Reply:
resolved by b99f9c8